extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -51,7 +51,9 @@ function keyByProductTagId(config) {
 export function storeShoppingConfig(pageElement, config) {
   const win = pageElement.ownerDocument.defaultView;
   return Services.storyStoreServiceForOrNull(win).then((storeService) => {
-    storeService?.dispatch(Action.ADD_SHOPPING_DATA, config);
+    const shoppingConfig = {};
```

Comment:
Nit: maybe we could name this something like `pageIdToConfig` because the "shoppingConfig" and "config" variables have similar names

Reply:
sure